server/request_processor.go

```diff
@@ -330,7 +337,9 @@ func (r *RpcRequest) handleCancelTx() (requestCompleted bool) {
 	}
 
 	cancelPrivTxArgs := flashbotsrpc.FlashbotsCancelPrivateTransactionRequest{TxHash: initialTxHash}
-	_, err = FlashbotsRPC.FlashbotsCancelPrivateTransaction(r.relaySigningKey, cancelPrivTxArgs)
+
+	fbRpc := flashbotsrpc.New(r.relayUrl)
```

Comment:
does recreating the flashbotsrpc instance have negative performance impact, like not reusing the HTTP client connection pool?